ca/certificates.go

```diff
@@ -209,23 +200,56 @@ func (rca *RootCA) RequestAndSaveNewCertificates(ctx context.Context, paths Cert
 		return nil, err
 	}
 
-	// Ensure directory exists
-	err = os.MkdirAll(filepath.Dir(paths.Cert), 0755)
+	var kekUpdate *KEKUpdate
+	for i := 0; i < 5; i++ {
+		kekUpdate, err = rca.getKEKUpdate(ctx, X509Cert, tlsKeyPair, r)
```

Comment:
This section is still trying 5 times, and then bailing.  As mentioned before, `RenewTLSConfig` keeps retrying based on the expiry.
We now no longer fail if it goes from an unlocked cluster to a locked cluster - if it fails, we'll depend on the `RenewTLSConfig` to eventually re-download.